app/helpers/kaui/subscription_helper.rb

```diff
@@ -101,9 +101,9 @@ def humanized_subscription_cancelled_date(sub, account)
       else
         cancelled_date = format_date(sub.cancelled_date, account.time_zone).html_safe
         if Time.parse(sub.cancelled_date) > current_time(account.time_zone)
-          'Pending cancellation on ' + cancelled_date
+          'Entitlement date: ' + cancelled_date + ' [PENDING]'
```

Comment:
@sbrossie Should we still distinguish the `PENDING` case or simply display `Entitlement date: XXX `? Not quite sure why the entitlement date formatting is different than the billing one. 😕

Reply:
@pierre @wwjbatista Entitlement date formatting should be similar to Billing date formatting. Either we append `[PENDING]` -- for both -- when date is in the future or we drop this altogether. Thoughts?

Reply:
I would drop the `[PENDING]`.

Reply:
I will remove the `[PENDING]` part.